source/text/implementation/vss-strings-buffers.adb

```diff
@@ -69,7 +69,10 @@ package body VSS.Strings.Buffers is
       if Handler /= null then
          Handler.Unreference (Self.Data);
 
-         Self.Data := (others => <>);
+         Self.Data := (In_Place => False,
```

Comment:
Old construction here is quite important for maintainability (and performance), it means that everything is initialized by default if necessary.

Reply:
not sure I understand this comment: for maintainability, the compiler will tell us if there's a field we omitted, right? And for initializing things to default, I think specifying values will work more reliably than "others => <>", which actually doesn't perform initialization.

Regarding performance I don't understand how the new syntax is worse. Is it better to declare a constant for the "Default initialized" value and copy this constant here?